lib/components/Widgets/WidgetContainer/index.tsx

```diff
@@ -1,48 +1,39 @@
+import { CardDescription } from "@/factorial-one"
 import {
   Card,
   CardContent,
-  CardDescription,
-  CardFooter,
   CardHeader,
+  CardInfo,
+  CardLink,
+  CardSubtitle,
   CardTitle,
 } from "@/ui/card"
-import { TrendingUp } from "lucide-react"
 import { forwardRef, ReactNode } from "react"
 
 export interface WidgetContainerProps {
   header: {
     title: string
-    description: string
-  }
-  footer?: {
-    trend: string
-    time: string
+    subtitle?: string
+    info?: string
+    link?: string
   }
+  description?: string
 }
 
 export const WidgetContainer = forwardRef<
   HTMLDivElement,
   WidgetContainerProps & { children: ReactNode }
->(({ header, footer, children }, ref) => (
-  <Card ref={ref}>
+>(({ header, description, children }, ref) => (
+  <Card ref={ref} className="min-h-[200px]">
     <CardHeader>
-      <CardTitle>{header.title}</CardTitle>
-      <CardDescription>{header.description}</CardDescription>
+      <div className="flex grow flex-row gap-1.5">
```

Comment:
I think this `flex` needs `items-center` to keep things aligned in the vertical axis.

Reply:
Done!